### PR TITLE
contributors: update command help

### PR DESCRIPTION
**Pull request by zz-jason**

### CODE REVIEW

**Comment by zz-jason** [comment]
@zhouqiang-cl PTAL

**Comment by zz-jason** [comment]
@gingerkidney PTAL

File: contributors/command-help.md [hunk 1]
```diff
@@ -15,22 +15,50 @@ Supported Repositories:
...
+* `idc-jenkins-ci-tidb/check_dev`
+* `idc-jenkins-ci-tidb/check_dev_2`
 
 ### `/run-all-tests`
 
 #### Description
 
-Triggers all the CI test cases.
+Triggers all the CI test cases. Here is the list of all the CI test cases:
+
```
Comment (zhouqiang-cl) [inline contributors/command-help.md]: /run-all-tests will not run build/build_check_race/check_dev/check_dev_2
and only tidb repo need mybatis-test/sqllogic-test etc. did it ok?

Comment (zz-jason) [reply contributors/command-help.md]: Addressed, PTAL again. Thanks.

**Comment by gingerkidney** [comment]
LGTM

**Pull request merged**